force-app/main/default/classes/MaintenanceRequestHelper.cls

```diff
@@ -0,0 +1,57 @@
+public with sharing class MaintenanceRequestHelper {
+
+    public static void createMaintenanceRequestRecord(List<Case> caseList) {
+        try{
+            List<Case> newCases = new List<Case>();
+            Integer cycle = getMaintenanceCycle(caseList) == null ? 0 : getMaintenanceCycle(caseList);
+            for(Case c: caseList) {
```

Comment:
please name variable in more meaningful way even inside loop, e.g. "caseLoop" or "item" or simply "case" but "c" is bad var name